jjb/global-pipelines/edgex-global-pipelines.yaml

```diff
@@ -0,0 +1,15 @@
+---
+
+- project:
+    name: edgex-global-pipelines
+    project-name: edgex-global-pipelines
+    project: edgex-global-pipelines
+    mvn-settings: EdgeX
```

Comment:
The "EdgeX" settings file is not a maven file. What are you trying to get it to provide?

Reply:
Yes I tested this on the sandbox...I will remove the mvn-settings file.

Reply:
I have requested maven settings on service desk

Reply:
Sandbox job:
https://jenkins.edgexfoundry.org/sandbox/blue/organizations/jenkins/edgex-global-pipelines-verify-pipeline/detail/edgex-global-pipelines-verify-pipeline/1/pipeline